pkg/appStore/deployment/fullMode/AppStoreDeploymentFullModeService.go

```diff
@@ -297,3 +303,106 @@ func (impl AppStoreDeploymentFullModeServiceImpl) createInArgo(chartGitAttribute
 
 	return nil
 }
+
+func (impl AppStoreDeploymentFullModeServiceImpl) UpdateValuesYaml(installAppVersionRequest *appStoreBean.InstallAppVersionDTO) (*appStoreBean.InstallAppVersionDTO, error) {
+	acdAppName := fmt.Sprintf("%s-%s", installAppVersionRequest.AppName, installAppVersionRequest.EnvironmentName)
+	gitOpsRepoName := installAppVersionRequest.GitOpsRepoName
+	if len(gitOpsRepoName) == 0 {
```

Comment:
i think we are using similar code to fetch gitops repo at other places also.. should extact it out as function

Reply:
done